modules/extension/xsd/xsd-gml3/src/main/java/org/geotools/gml3/bindings/PolygonTypeBinding.java

```diff
@@ -98,15 +99,38 @@ public Class getType() {
     public Object parse(ElementInstance instance, Node node, Object value)
         throws Exception {
         //TODO: schema allows no exterior ring, but what the heck is that all about ?
-        LinearRing exterior = (LinearRing) node.getChildValue("exterior");
+        Object o = node.getChildValue("exterior");
+
+        LinearRing exterior = null;
+        if(o instanceof String){
```

Comment:
Yes, I figured it could be something like that, but I was a bit lost on where that should be fixed. Will try monday to figure it out where the more correct place would be. 
Thanks!